src/Lykke.blue.Service.ReferralLinks/Controllers/ReferralLinksController.cs

```diff
@@ -397,6 +397,11 @@ private string ValidateClaimRefLinkAndRequest(IReferralLink refLink, ClaimReferr
                 {
                     var newRefLinkClaimSender = await CreateNewRefLinkClaim(refLink, refLink.SenderClientId, true, false);
 
+                    if (Math.Abs(refLink.Amount) < MinimalAmount)
+                    {
```

Comment:
Please log a warning for this situation. Since we have similar validation for creating ref links, it points to a corrupted data.